pages/contributing/metadata_guidelines.md

```diff
@@ -2,3 +2,185 @@
 title: Metadata Guidelines
 summary: Descriptions of different metadata schemas used to describe various resources and pages in RSQKit
 ---
+
+In this page we describe the metadata schemas  used to describe various resources and pages in the RSQKit (adapted from the [RDMKit page metadata guide](https://rdmkit.elixir-europe.org/page_metadata)).
+
+## Metadata attributes of a page
+When creating a new page in RSQKit, the following metadata attributes may be edited:
+
+* `title`: Specify here the title of the page. This wil be the H1 title (replacing the top level title using the # in markdown )
+
+* `summary`: Using this attribute it is possible to specify a summary which will be displayed under the title of the page. This summary will also be used as description of your page when the page is tagged.
+
+* `description`: Short sentence about the page starting with a lowercase. This sentence is visualized when pages are automatically listed using a tag.
+
+* `contributors`: list here all the contributors that helped in establishing the page. This will be the full name of the person. Make sure that the person name that is listed can be found in the CONTRIBUTORS.yaml file in the _data directory if you want to link the github id and other contact information (see below for contributor metadata).
```

Comment:
```suggestion
* `contributors`: list here all the contributors that helped in establishing the page. This will be the full name of the person. Make sure that the person name that is listed can be found in the CONTRIBUTORS.yml file in the _data directory if you want to link the github id and other contact information (see below for contributor metadata).
```